### PR TITLE
docs: updating upgrade instructions for v1.2.0

### PR DESCRIPTION
**Pull request by TheMarstonConnell**

### CODE REVIEW

File: upgrades/gov-props/v1.2.0.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+* Re-added Notification Module
 * Added Oracle Module
 * Fixed RNS marketplace bugs
 * Increased Code Test Coverage to ~50%
 * Fixed Go Package & Proto Layout
 * Removed Ignite CLI as a dependency & fixed codegen bugs associated with it
 * Improved Safety of Storage/Filetree modules from v1.1.1
-* Updated Cosmwasm Version from v0.29.1 -> v0.29.2
-* Updated CosmosSDK Version from v0.45.10 -> v0.45.14
+* Updated Cosmwasm Version from v0.29.1 -> v0.30.0 w/ Patch
```
Comment (BiPhan4) [inline upgrades/gov-props/v1.2.0.md:23]: Master branch is only at Cosmwasm v0.29.2-patch.1. Do we plan on making this update in the future? @TheMarstonConnell

File: upgrades/gov-props/v1.2.0.md [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
+* Updated CosmosSDK Version from v0.45.10 -> v0.45.11
 * Updated IBC Version from v3.3.1 -> v3.4.0
 * Updated Tendermint Version from Tendermint v0.34.22 -> [Informal's Tendermint](https://github.com/tendermint/tendermint/issues/9972) v0.34.23 
 * Updated Security of Storage Module proof mechanism from using Blake2b to Sha3-512
 * Refactored HTTP server into new repo, see [canine-provider](https://github.com/JackalLabs/canine-provider)
 
-For a full list of changes, please visit https://github.com/JackalLabs/canine-chain/compare/v1.1.2-hotfix...v1.2.0-beta.7
+For a full list of changes, please visit https://github.com/JackalLabs/canine-chain/compare/v1.1.2-hotfix...v1.2.0-rc.1
```
Comment (BiPhan4) [inline upgrades/gov-props/v1.2.0.md:30]: This link brings up a page with message 'There isn’t anything to compare.'

**Review by BiPhan4 — CHANGES_REQUESTED** [review]
Possible changes?

File: upgrades/gov-props/v1.2.0.md (hunk 2 above)
Comment (TheMarstonConnell) [reply upgrades/gov-props/v1.2.0.md:30]: Yeah it's future-proofing a release

File: upgrades/gov-props/v1.2.0.md (hunk 1 above)
Comment (TheMarstonConnell) [reply upgrades/gov-props/v1.2.0.md:23]: Oh I thought it was at 30, yeah I'll roll it back

**Review by BiPhan4 — APPROVED** [review]
LGTM!

**Review by BiPhan4 — APPROVED** [review]
Confirmed changes I requested. LGTM!

**Pull request merged**